src/lib/provable/field.ts

```diff
@@ -670,14 +667,12 @@ class Field {
 
   /**
    * Assert that this {@link Field} is less than another "field-like" value.
-   * Calling this function is equivalent to `Field(...).lessThan(...).assertEquals(Bool(true))`.
+   *
+   * Note: This uses fewer constraints than `x.lessThan(y).assertTrue()`.
```

Comment:
didn't like that this said it "was equivalent", people should use the assertion version if possible because they are cheaper